CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ A more detailed list of changes is available in the corresponding milestones for
 #### On the Google Fonts profile
   - **[com.google.fonts/check/font-v]:** Change rationale to reflect the fact that this check emits an INFO (issue #4067)
   - **[com.google.fonts/check/vertical_metrics_regressions]:** Fix an error when the provided font did not have a Regular style. (issue #3897)
+  - **[com.google.fonts/check/mandatory_glyphs]:** The check which ensures that the `.notdef` glyph is not empty has been upgraded from a WARN to a FAIL. (issue #4093)
```

Comment:
This is not a change in Google Fonts profile, but Universal profile instead. I'll fix this now.